### PR TITLE
update(bootstrap): v5.1 updates

### PR DESCRIPTION
**Pull request by peterblazejewicz**
- unify `BaseComponent.getInstance` api to align wit 5.1 interface
- apply minor cleanup
- apply DT formatt

https://github.com/twbs/bootstrap/releases/tag/v5.1.0

Thanks!

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).
- [x] Provide a URL to documentation or source code which provides context for the suggested changes
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/bootstrap/js/dist/button.d.ts [hunk 1]
```diff
@@ -11,9 +11,8 @@ declare class Button extends BaseComponent {
      */
     dispose(): void;
```
Comment (martin-badin) [inline types/bootstrap/js/dist/button.d.ts:12]: The base component includes the function. Please remove it.

**Review by martin-badin — CHANGES_REQUESTED** [review]

File: types/bootstrap/js/dist/button.d.ts (hunk 1 above)
Comment (peterblazejewicz) [reply types/bootstrap/js/dist/button.d.ts:12]: amended, thx!

**Review by martin-badin — APPROVED** [review]
Thanks

**Comment by peterblazejewicz** [comment]
Ready to merge 💘

File: types/bootstrap/js/dist/button.d.ts (hunk 1 above)
Comment (arnil6585) [reply types/bootstrap/js/dist/button.d.ts:12]: cd2bd01e35b4197fdd2f3a89673293ecf7d5bf94

**Comment by arnil6585** [comment]
8fc59c67996279c8d1dc0f34176ad9a59e38727d

**Review by arnil6585 — APPROVED** [review]

**Pull request merged**